livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -888,7 +899,13 @@ def _handle_error(self, event: ErrorEvent) -> None:
         )
         self.emit(
             "error",
-            llm.ErrorEvent(type=event.error.type, message=event.error.message),
+            llm.RealtimeModelError(
+                type="realtime_model_error",
+                timestamp=time.time(),
+                label=self._realtime_model._label,
+                error=event.error,
```

Comment:
let's be consistent and give an Exception here, let's use APIError? 
we may not have all the details, but I think it's fine for now